cpp/test/Ice/ami/AllTests.cpp

```diff
@@ -1099,7 +1088,7 @@ allTests(Test::TestHelper* helper, bool collocated)
             {
                 test(p->opBatchCount() == 0);
                 auto b1 = Ice::uncheckedCast<Test::TestIntfPrx>(
-                    p->ice_getConnection()->createProxy(p->ice_getIdentity())->ice_batchOneway());
+                    p->ice_getConnection()->createProxy(p->ice_getIdentity()))->ice_batchOneway();
```

Comment:
is there a better way to create a fixed proxy?

Reply:
Use `ice_fixed`?

Reply:
There is no reason to use uncheckedCast for anything. If you have an ObjectPrx proxy, you can create a typed proxy with the "const ObjectPrx&" ctor, as in:
```cpp
Test::TestInftPrx b1(p->ice_getConnection()->createProxy(p->ice_getIdentity()));
```

Reply:
Here, I would indeed use ice_fixed:
```cpp
auto b1 = p->ice_fixed(p->ice_getConnection())->ice_batchOneway();
```